File: pta/pat_a/1027.py
import re
i=input()
i=i.split(" ")
di=['0','1','2','3','4','5','6','7','8','9','A','B',"C"]
def to13(n):
    gw=n%13
    sw=n//13
    return di[sw]+di[gw]

print("#{}{}{}".format(to13(int(i[0])),to13(int(i[1])),to13(int(i[2]))))

File: pta/pat_a/1027.cpp
#include <iostream>

using namespace std;

string dict[]={"0","1","2","3","4","5","6","7","8","9","A","B","C"};

string to13(int n){
    int gw=n%13;
    int sw=n/13;
    return dict[sw]+dict[gw];
}

int main()
{
    string a="123""456";
    cout<<a<<endl;
    return 0;
    int r,g,b;
    cin>>r>>g>>b;
    cout<<"#"<<to13(r)<<to13(g)<<to13(b)<<endl;
    return 0;
}